### PR TITLE
Post-processor metric parser

### PR DESCRIPTION
**Pull request by niclashedam**
This PR implements a processor, that may take a run.log result at transform it to a structured json file with the metrics. I have implemented a fio filter as an example.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The following people have contributed to the implementation of CIJOE:
 * Hans Holmberg, [CNEX Labs, Inc.]
 * Javier González, [CNEX Labs, Inc.]
 * Weifeng Guo, [CNEX Labs, Inc.]
+* Niclas Hedam, [IT University of Copenhagen]
```
Comment (safl) [inline CONTRIBUTORS.md]: Please add this to its own commit, e.g. "contributors: added 'Niclas Hedam'"

File: bin/cij_processor [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        if val is None:
+            cij.err(
+                "post:post: failed parsing environment variable 'CIJ_%s'" %
+                key
+            )
+            return None
+
+    # Parse the Command-Line
+    prsr = argparse.ArgumentParser(
+        description='cij_processor - CIJOE Post Processor',
```
Comment (safl) [inline bin/cij_processor]: Describe it as: "cij_processor - CIJOE Test Result Processor"

**Comment by niclashedam** [comment]
The binary descriptor has been updated and the update of contributors list put into its own commit.

**Comment by safl** [comment]
Thanks!

I have picked the contributor-commit and applied it on ``master`` and ``rework``.
If you rebase this PR on ``rework``, and set it the PR to target ``rework`` then I'll pull it in.
If you don't then I'll just pick it and close the PR.

Let me know :)

**Comment by safl** [comment]
Picked on ``rework`` branch with an added ``TODO`` note in commit-message.

**Pull request closed without merging**